### PR TITLE
updating edit form

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ module:
   ckeditor: 0
   color: 0
   comment: 0
+  comments_order: 0
```
Comment (ElXyfer) [inline config/sync/core.extension.yml:17]: Why is this here?

Comment (katlazarova) [reply config/sync/core.extension.yml]: It's not

File: web/modules/custom/pineapple_user/pineapple_user.module [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    $form['account']['name']['#weight'] = 1;
+    $form['account']['roles']['#weight'] = 5;
+  }
+}
+
+/**
+ * Implements hook_element_info_alter().
+ */
+function pineapple_user_element_info_alter(&$info) {
+  $v = 0;
```
Comment (ElXyfer) [inline web/modules/custom/pineapple_user/pineapple_user.module:37]: Remove debugging code.

File: web/modules/custom/pineapple_user/pineapple_user.module [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  if (isset($info['password_confirm'])) {
+    $info['password_confirm']['#process'][] = '_pineapple_user_change_title';
+  }
+}
+
+function _pineapple_user_change_title(&$element) {
+  $element['pass1']['#title'] = t('New Password');
```
Comment (ElXyfer) [inline web/modules/custom/pineapple_user/pineapple_user.module:44]: Confirm password field should probably be renamed to "Confirm new password"

File: web/modules/custom/pineapple_user/pineapple_user.module (hunk 2 above)
Comment (katlazarova) [reply web/modules/custom/pineapple_user/pineapple_user.module]: Done.

File: web/modules/custom/pineapple_user/pineapple_user.module (hunk 3 above)
Comment (katlazarova) [reply web/modules/custom/pineapple_user/pineapple_user.module:43]: Done.
